src/main/java/org/jabref/gui/preferences/XmpPrefsTab.java

```diff
@@ -107,35 +99,20 @@ public XmpPrefsTab(JabRefPreferences prefs) {
 
     }
 
+    @Override
     public Node getBuilder() {
         return builder;
     }
 
-    public static class TableRow {
-        private SimpleStringProperty name;
-
-        TableRow(String name) {
-            this.name = new SimpleStringProperty(name);
-        }
-
-        public void setName(String name) {
-            this.name.set(name);
-        }
-
-        public String getName() {
-            return name.get();
-        }
-    }
-
     /**
      * Load settings from the preferences and initialize the table.
      */
     @Override
     public void setValues() {
-        tableRows.clear();
-        tableRows.addAll(data);
-        privacyFilterCheckBox.setSelected(JabRefPreferences.getInstance().getBoolean(
-                JabRefPreferences.USE_XMP_PRIVACY_FILTER));
+        List<XMPPrivacyFilter> xmpExclusions = prefs.getStringList(JabRefPreferences.XMP_PRIVACY_FILTERS).stream().map(XMPPrivacyFilter::new).collect(Collectors.toList());
+        fields.clear();
```

Comment:
I'm not sure, but I thought there is a `setAll` method (instead of clear + addAll)